syntax ="proto3";
package fixbug;

option cc_generic_services=true;
message ResultCode
{
    int32 errcode=1;
    bytes errmsg=2;
}

message LoginResult
{
    bytes name =1;
    bytes pwd = 2;
}

message LoginResponse
{
    ResultCode result=1;
    bool success=2;
}

service UserServiceRpc
{
    rpc Login(LoginResult)  returns(LoginResponse);
}